src/components/WelcomeStepsModal/WelcomeModalPostalCode.tsx

```diff
@@ -96,8 +96,9 @@ export const WelcomeModalPostalCode: React.FC = () => {
       <div style={{ display: "flex", flexDirection: "row", alignItems: "center", marginBottom: "10px" }}>
         <Checkbox value={checked} checked={checked} onChange={handleCheckedChange} />
         <Typography className={classes.smallText} style={isCheckboxError ? { color: "red" } : {}}>
-          Ja, ich habe die <Link to={WelcomeModalStep.StepPostalCodeDataPrivacy}>Datenschutzerklärung</Link> zur
-          Kenntnis genommen und willige ein.
+          <Trans t={t} i18nKey="welcome.regional-risk.accept-privacy">
```

Comment:
Not sure why we have to pass `t` explicitly here. If anybody has an idea, please let me know. 🙂